include/gdf/cffi/types.h

```diff
@@ -1,3 +1,6 @@
+#ifndef _GDF_TYPES_H_
```

Comment:
Same issue as above in `gdf/cffi/functions.h`

Reply:
ok we will drop that